test/Microsoft.AspNetCore.Razor.Language.Test/Legacy/CSharpDirectivesTest.cs

```diff
@@ -1957,7 +1957,7 @@ public void ParseAddOrRemoveDirective_CreatesErrorIfInvalidLookupText_DoesNotThr
             Block expected,
             params RazorDiagnostic[] expectedErrors)
         {
-            var result = ParseCodeBlock(document, descriptors, designTime: false);
+            var result = ParseCodeBlock(RazorLanguageVersion.Latest, document, descriptors, designTime: false);
```

Comment:
Most of the changes here were just to thread the version through all of the various tests